fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java

```diff
@@ -486,6 +500,68 @@ public Type postgresqlTypeToDoris(JdbcFieldSchema fieldSchema) {
         }
     }
 
+    public Type clickhouseTypeToDoris(JdbcFieldSchema fieldSchema) {
+        String ckType = fieldSchema.getDataTypeName();
+        if (ckType.startsWith("LowCardinality")) {
+            ckType = ckType.substring(15, ckType.length() - 1);
+        }
+        if (ckType.startsWith("Nullable")) {
+            ckType = ckType.substring(9, ckType.length() - 1);
+        }
+        if (ckType.startsWith("Decimal")) {
```

Comment:
Should be use `if else` for these conditions?

Reply:
done